garminconnect/__init__.py

```
@@ -716,14 +716,56 @@ def get_personal_record(self) -> Dict[str, Any]:
+        return self.connectapi(url)
+
+    def get_in_progress_badges(self) -> List[Dict[str, Any]]:
+        """Return in progress badges for current user."""
+
+        logger.debug("Requesting in progress badges for user")
+
+        earned_badges = self.get_earned_badges()
+        available_badges = self.get_available_badges()
+
+        # Filter out badges that are not in progress
+        def badge_in_progress(badge):
```

Ruff (0.11.9)

Missing return type annotation for private function `badge_in_progress`

(ANN202)